tutorials/platform/android/native_debug_symbols.rst

```diff
@@ -0,0 +1,75 @@
+.. _doc_native_debug_symbols:
+
+Native Debug Symbols
+====================
+
+When your game crashes on Android, you often see obfuscated stack traces in Play Console or other crash reporting tools like Firebase Crashlytics.
+To make these stack traces human-readable (symbolicated), you need native debug symbols that correspond to your game’s exported build.
+
+Godot now provides downloadable native debug symbols for each official export template. However, For custom template
+you'd have to generate those when building your template.
+
+Getting the Native Debug Symbols
+--------------------------------
+
+There are two ways to obtain native debug symbols, depending on whether you are using the official export templates or building custom templates.
+
+1. Getting Native Debug symbols for Official template
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Native debug symbol files are provided for every stable Godot release and can be downloaded from the `GitHub release page <https://github.com/godotengine/godot/releases/>`_.
+
+For example, to get the native debug symbols for version ``4.5.1.stable``:
+
+- Go to `4.5.1.stable release page <https://github.com/godotengine/godot/releases/>`_
+- Download the release artifact ``Godot_native_debug_symbols.4.5.1.stable.template_release.android.zip``
+
+2. Getting Native Debug symbols for custom build
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Your exported template and its native debug symbols must come from the **same build**, so you can use the official symbols only if you are using the **official export templates**.
+If you are building **custom export templates**, you need to generate matching symbol files yourself.
+
+To do so, add ``debug_symbols=yes separate_debug_symbols=yes`` to your scons build command.
+This will generate a file named ``android-template-release-native-symbols.zip`` containing the native debug symbols for your custom build.
+
+For example,
+
+::
+
+    scons platform=android target=template_release debug_symbols=yes separate_debug_symbols=yes generate_android_binaries=yes
```

Comment:
We should mention that if they're building for multiple archs, then the scons parameters to generate the native debug symbols must come last.

See https://github.com/godotengine/godot-build-scripts/blob/main/build-android/build.sh#L40-L43